catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc

```diff
@@ -0,0 +1,373 @@
+provides results based on the defined criteria and detailed information about each result.
+Additionally, a user can save individual records that were returned by the query to a workspace, so
+they can be referenced in the future. The user can also save the search criteria to a workspace so
+the query can be run again.
+
+==== Search Criteria
+
+The Search UI queries a Catalog using the following criteria.
+
+[cols="1,6", options="header"]
+|===
+|Criteria
+|Description
+
+|Text
+|Search by free text using the grammar of the underlying endpoint/Catalog.
+
+|Time
+|Search based on relative or absolute time of the created, modified, or effective date.
+
+|Location
+|Search by latitude/longitude or the USNG using a point-radius or bounding box.
+
+|Type
+|Search for specific content types.
+
+|Sorting
+|Sort results by relevance, distance, created time, modified time or effective time.
+
+|Additional Sources
+|Select *All Sources* or *Specific Sources*.
+
+|All Sources
+|Create an enterprise search. All federations are queried.
+
+|Specific Sources
+|Search a specific source(s). If a source is unavailable, it is displayed in red text.
+
+|===
+
+image:search-form.png[Search criteria]
+
+==== Results
+
+After a query is executed, the records matching the search criteria are automatically displayed in
+the Results pane.
+
+[cols="1,6", options="header"]
+|===
+
+|Item
+|Description
+
+|image:result-status-toggle.png[Search criteria]
+|Enhanced search toggle. Enables the enhanced search menu (see below). Allows users to filter and
+ refine search and build more sophisticated queries.
+
+|Results
+|The number of records that were returned as a result of the query. Only the top 250 results are
+ displayed, with the most relevant records displayed at the top of the list. If more than 250
+ results are returned, try narrowing the search criteria.
+
+|Search button
+|Navigates back to the Search pane.
+
+|Save button
+|Allows the user to select individual records to save.
+
+|Records list
+|Shows the results of the search. The following information is displayed for each record:
+
+Title – The title of the record is displayed in blue text. Select the title of the record to view
+more details.
+
+Source – The gray text displayed below the record title is the data source (e.g., ddf.distribution)
+and the amount of time that has passed since the record was modified (e.g., an hour ago).
+
+|===
+
+image:results-list.png[Search results]
+
+==== Enhanced Search
+
+The enhanced search menu allows more granular filtering of results and the ability to construct
+sophisticated queries.
+
+[cols="1,6", options="header"]
+|===
+
+|Item
+|Description
+
+|Source
+|List of all sources searched with check boxes to allow users to refine searches to the most
+ relevant sources.
+
+|Metadata Content Type
+|List of metadata content types found in the search, with the ability to select and deselect
```

Comment:
Technically this may not be worth changing, but useful FYI: no comma is needed here: `...in the search, with the ability...`; we may take a natural pause when speaking this but it's not a run-on sentence. Once again, mainly pointing out for future reference. I also could be wrong.